tests/foreman/api/test_registration.py

```diff
@@ -227,3 +227,49 @@ def test_negative_global_registration_without_ak(module_target_sat):
     with pytest.raises(HTTPError) as context:
         module_target_sat.api.RegistrationCommand().create()
     assert 'Missing activation key!' in context.value.response.text
+
+
+def test_positive_capsule_without_registration_module(
+    module_target_sat,
+    module_capsule_configured,
+    module_ak_with_cv,
+    rhel8_contenthost,
+    module_entitlement_manifest_org,
+    module_location,
+):
+    """Test a capsule without registration module
+
+    :id: a2f23e42-648d-4428-a961-6e0b933c6dff
+
+    :steps:
+
+            1. Get a configured capsule
+            2. The registration is set to False on capsule by default
+            3. Try to register host with that capsule
+
+    :expectedresults: Registration fails with HTTP error code 422 and an error message.
+    """
+    org = module_entitlement_manifest_org
+    module_target_sat.api.SmartProxy().search(
+        query={'search': f'name = {module_capsule_configured.hostname}'}
+    )[0]
+
+    nc = module_capsule_configured.nailgun_smart_proxy
+    module_target_sat.api.SmartProxy(id=nc.id, organization=[org]).update(['organization'])
+    module_target_sat.api.SmartProxy(id=nc.id, location=[module_location]).update(['location'])
+
+    res = module_capsule_configured.install(
+        cmd_args={},
+        cmd_kwargs={'foreman-proxy-registration': 'false', 'foreman-proxy-templates': 'true'},
+    )
+    assert res.status == 0
+
+    with pytest.raises(HTTPError):
```

Comment:
Can we define and assert error message here to complete this negative test?
```suggestion
    with pytest.raises(HTTPError, match=f'{error_message}'):
```

Reply:
Updated.